ui/src/logic/useSidebarSort.ts

```diff
@@ -8,7 +8,7 @@ import useAllBriefs from './useAllBriefs';
 
 export const ALPHABETICAL = 'A → Z';
 export const DEFAULT = 'Arranged';
-export const RECENT = 'Recent';
```

Comment:
This change will mean that users who had the sidebar set to "Recent" before this update will need to manually set it again after the update. It will show "Recent Activity" as the selected sort mode, but it will actually be alphabetical. Probably best to just change the way this is displayed where it's being displayed, rather than with this change to the constant that we use in the logic/persistence.

Or, account for the old name being in the store and translate it to the new name.